apps/desktop/electron-builder-win.config.js

```
@@ -0,0 +1,30 @@
+    '**/node_modules/@stoprocent/bluetooth-hci-socket/**',
+  ],
+
+  nsis: {
+    oneClick: false,
+    installerSidebar: 'app/build/static/images/icons/installerSidebar.bmp',
+    deleteAppDataOnUninstall: true,
+  },
+  win: {
+    extraResources: [
+      {
+        from: 'app/build/static/bin/bridge/win-${arch}',
```

ESLint

[error] Unexpected template string expression.

(no-template-curly-in-string)